ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -0,0 +1,411 @@
+              {!isAddInstitution && errors && errors.displayName}
+              </div>
+            <label style={styles.label}>Institution Type</label>
+            <Dropdown style={{width: '16rem'}} data-test-id='role-dropdown'
+                      placeholder='Your Role'
+                      options={this.institutionTypeOptions}
+                      value={institution.organizationTypeEnum}
+                      onChange={v => this.updateInstitutionRole(v.value)}/>
+            <div style={{color: colors.danger}}>{!isAddInstitution && errors && errors.organizationTypeEnum}</div>
+
+            {showOtherInstitution && <TextInputWithLabel value={institution.organizationTypeOtherText}
+               onChange={v => this.setState(fp.set(['institution', 'organizationTypeOtherText'], v))}
+               inputStyle={{width: '16rem', marginTop: '0.8rem'}}/>}
+            <label style={styles.label}>Agreement Type</label>
+            <Dropdown style={{width: '16rem'}} data-test-id='agreement-dropdown'
+                      placeholder='Your Agreement'
+                      options={this.institutionAgreementTypeOptions}
+                      value={institution.duaTypeEnum}
+                      onChange={v => this.setState(fp.set(['institution', 'duaTypeEnum'], v.value))}/>
+            {institution.duaTypeEnum === DuaType.RESTRICTED && <FlexColumn data-test-id='emailAddress' style={{width: '16rem'}}>
+              <label style={styles.label}>Accepted Email Address</label>
+              <TextArea value={institution.emailAddresses && institution.emailAddresses.join(',\n')}
+                        data-test-id='emailAddressInput'
+                        onBlur={(v) => this.validateEmailAddresses()}
+                  onChange={(v) => this.setEmailDomain(v, 'emailAddresses')}/>
+              {this.state.invalidEmailAddress && <div data-test-id='emailAddressError' style={{color: colors.danger}}>
+                {this.state.invalidEmailAddressMsg}
+                </div>}
+            </FlexColumn>}
+            {institution.duaTypeEnum === DuaType.MASTER && <FlexColumn data-test-id='emailDomain' style={{width: '16rem'}}>
+              <label style={styles.label}>Accepted Email Domain</label>
+              <TextArea value={institution.emailDomains && institution.emailDomains.join(',\n')} onBlur={(v) => this.validateEmailDomain()}
+                        data-test-id='emailDomainInput'
+                        onChange={(v) => this.setEmailDomain(v, 'emailDomains')}/>
+              {this.state.invalidEmailDomain && <div data-test-id='emailDomainError' style={{color: colors.danger}}>
+                {this.state.invalidEmailDomainsMsg}
+                </div>}
+            </FlexColumn>}
+          </FlexColumn>
+          <FlexColumn style={{width: '50%', marginRight: '1rem'}}>
+            <label style={{...styles.label, marginTop: '0rem'}}>User instruction (Optional)</label>
```

Comment:
Plural: instructions

Maybe something more explicit would be appropriate, like "User Email Instructions Text"